tests/l2/dgpu/README.md

```diff
@@ -87,5 +87,168 @@ $ oc logs intel-dgpu-hwinfo-44k4d
   Attached to: #89 (PCI bridge)
 ```                        
 
+### Verify Media support for Intel® Data Center GPU
```

Comment:
Verify the Media feature provisioning for Intel® Data Center GPU

Reply:
Done